docs/command_line_reference.rst

```diff
@@ -41,68 +44,99 @@ command above).
-:``--reregenerate``: regenerates, reconfigures, recompiles and reinstalls the build.
+:--checkout:      Deletes the local Git repository and clones it.
+:--compile:       Compiles the build.
+:--configure:     Configures the build.
+:--delete-source: Deletes the source directory when purging.
+:--install:       Installs the build files.
+:--purge:         Deletes all related build files.
+:--recheckout:    Deletes the cloned git repository, re-clones, regenerates,
+                  reconfigures, re-compiles, and reinstalls it.
+:--recompile:     Re-compiles and reinstall the build.
+:--reconfigure:   Re-configures the build.
+:--regenerate:    Re-generates the build.
+:--reinstall:     Re-installs the build.
+:--reregenerate:  Re-generates, re-configures, re-compiles, and reinstalls the build.
 
 The ``--checkout``, ``--recheckout`` and ``--purge`` arguments further require the ``-f`` argument to confirm
 their actions.
 
 experiments
 -----------
-Responsible to check, execute and remove experiments.
-It supports the following actions:
 
-:info: displays all related instances, instance sets, variant axes and variants of experiments
-   on the command-line.
-:list: lists all the experiments.
-   Executed experiments are shown in green, failed ones are shown in red, running ones in
-   yellow, and the non executed ones in the default command-line color. With the argument
-   ``--detailed`` it will show every single run. With ``--compact`` all runs with the same
-   experiment will be grouped together. The ``--full`` option forces simexpal to display the
-   full experiment name.
-:launch: launches all the non executed experiments.
-:print: displays all experimental output, including error outputs, on the command-line.
-:purge: deletes the experimental data. To confirm this action it needs the ``-f`` argument.
-:kill: terminates jobs submitted to or started by Slurm. To confirm this action it needs
-    the ``-f`` argument.
+Responsible to check, execute and remove experiments. The experiments command
+accepts the following actions flags:
+
+:info:   Displays all related instances, instance sets, variant axes and variants
+         of experiments on the command-line.
+
+:list:   Lists all the experiments. Executed experiments are shown in green,
+         failed ones are shown in red, running ones in yellow, and the non
+         executed ones in the default command-line color. With the argument
+         ``--detailed`` it will show every single run. With ``--compact`` all
+         runs with the same experiment will be grouped together. The ``--full``
+         option forces simexpal to display the full experiment name.
+
+:launch: Launches all the non executed experiments.
+
+:print:  Displays all experimental output, including error outputs, on the command-line.
+
+:purge:  Deletes the experimental data. To confirm this action it needs the ``-f`` argument.
+
+:kill:   Terminates jobs submitted to or started by Slurm. To confirm this
```

Comment:
What happens if the command is used with other launchers than Slurm? I would presume these jobs would also get killed, right?

Reply:
Renamed to *the scheduler*. Thanks for the hint!